GPU/Workflow/src/GPUWorkflowSpec.cxx

```diff
@@ -89,6 +89,45 @@ using namespace o2::tpc;
 
 namespace o2::gpu
 {
+
+using ClusterGroupParser = o2::algorithm::ForwardParser<ClusterGroupHeader>;
+struct ProcessAttributes {
+  std::unique_ptr<ClusterGroupParser> parser;
+  std::unique_ptr<GPUO2Interface> tracker;
+  std::unique_ptr<GPUDisplayFrontend> displayFrontend;
+  std::unique_ptr<TPCFastTransform> fastTransform;
+  std::unique_ptr<TPCPadGainCalib> tpcPadGainCalib;
+  std::unique_ptr<TPCPadGainCalib> tpcPadGainCalibBufferNew;
+  std::unique_ptr<o2::tpc::CalibdEdxContainer> dEdxCalibContainer;
+  std::unique_ptr<o2::tpc::CalibdEdxContainer> dEdxCalibContainerBufferNew;
+  std::unique_ptr<o2::trd::GeometryFlat> trdGeometry;
+  std::unique_ptr<GPUO2InterfaceConfiguration> config;
+  int qaTaskMask = 0;
+  std::unique_ptr<GPUO2InterfaceQA> qa;
+  std::vector<int> clusterOutputIds;
+  unsigned long outputBufferSize = 0;
+  unsigned long tpcSectorMask = 0;
+  int verbosity = 0;
+  bool readyToQuit = false;
+  bool allocateOutputOnTheFly = false;
+  bool suppressOutput = false;
+  bool updateGainMapCCDB = true;
+  bool disableCalibUpdates = false;
+  o2::gpu::GPUSettingsTF tfSettings;
+};
+
+/// initialize TPC options from command line
+void initFunctionTPC(ProcessAttributes* processAttributes, const GPUSettingsO2& confParam);
+
+/// storing new calib objects in buffer
+void finaliseCCDBTPC(ProcessAttributes* processAttributes, gpuworkflow::Config const& specconfig, ConcreteDataMatcher& matcher, void* obj);
+
+/// asking for newer calib objects
+void fetchCalibsCCDBTPC(ProcessAttributes* processAttributes, gpuworkflow::Config const& specconfig, ProcessingContext& pc);
+
+/// storing the new calib objects by overwritting the old calibs
+void setCalibsTPC(ProcessAttributes* processAttributes);
```

Comment:
perhaps we can find a better name here, like `storeUpdatedCalibsTPCPtrs` instead of `setCalibsTPC` to make it clear what happens here.